shared/productTypes.ts

```diff
@@ -651,7 +651,6 @@ export const PRODUCT_TYPES: { [productKey in ProductTypeKeys]: ProductType } = {
 		groupedProductType: 'subscriptions',
 		allProductsProductTypeFilterString: 'GuardianPatron',
 		urlPart: 'guardianpatron',
-		legacyUrlPart: 'guardianpatron',
```

Comment:
having this legaclyUrlPart the same as urlPart caused an infinite redirect if you went to `/guardianpatron`